std_msgs/Header header
BedPose bed_pose
Obstacle[] obstacles
